pages/privacy.vue

```diff
@@ -0,0 +1,54 @@
+<style lang="scss">
+section p {
+  text-align: left;
+}
+section img {
+  max-width: 300px;
+}
+</style>
+
+<template>
+  <div>
+    <section class="page-header">
+      <div class="container">
+        <h1><strong>{{$lt('title')}}</strong></h1>
+      </div>
+    </section>
+
+    <section v-for="(section, id) in $lt('sections')" :id="id" :key="id">
+      <div class="container">
+        <img v-if="id === 'fftf'" src="~assets/images/fftf-dark.svg"
```

Comment:
@kenmickles We discussed this, but I'm putting it here for posterity: This image inclusion is a bit strange, but given that requiring images from yml files is tricky, and these images are unlikely to be edited often, this solution will be ok for the time being.